Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -31,6 +31,10 @@
         <SelectAlias value="Broodling"/>
         <AIEvaluateAlias value="Broodling"/>
     </CUnit>
+    <CUnit id="BroodlingDefault">
+        <!-- Override  -->
+        <SubgroupPriority value="5"/>
+    </CUnit>

```

Comment:
Instead of overriding this unit you could re-parent affected units to `AP_BroodlingDefault`